docs/appendix/educational-resources.md

```diff
@@ -143,4 +143,27 @@ A full list of all released blogs can be found below:
 - [Get to Know EDEX](https://www.unidata.ucar.edu/blogs/news/entry/awips-tips-get-to-know)
 - [EDEX Data Retention](https://www.unidata.ucar.edu/blogs/news/entry/awips-tips-edex-data-retention)
 - [Adding ECMWF Data to EDEX](https://www.unidata.ucar.edu/blogs/news/entry/awips-tips-adding-ecmwf-data)
-- [Ingesting GOES Satellite Data](https://www.unidata.ucar.edu/blogs/news/entry/awips-tips-ingesting-goes-satellite)
\ No newline at end of file
+- [Ingesting GOES Satellite Data](https://www.unidata.ucar.edu/blogs/news/entry/awips-tips-ingesting-goes-satellite)
+
+---
+
+<img src="../../images/youtube-logo.png" alt="YouTube logo" align="right">
```

Comment:
I think the path to the image is just 
../images/youtube-logo.png

Reply:
I know it seems like it should be that way, but with the way mkdocs builds the urls, that doesn't get you to the right place.  With only one '..' you get:
![image](https://user-images.githubusercontent.com/21114460/183963659-d8c9a9bf-f50d-42fe-ab36-5adae67d7113.png)

With the incorrect url of ...awips2/appendix/educational-resources/#youtube-channel-and-playlist

(It needs to go back, above appendix, still)

Reply:
It also is shown as broken the current way, but I guess in these 2 instances you are using html instead of markdown so maybe that's why it's different

![image](https://user-images.githubusercontent.com/65614396/183964638-de58fccd-3a53-4d5f-9306-bf982d452fb6.png)